api/column_family.cc

```diff
@@ -894,6 +894,30 @@ void set_column_family(http_context& ctx, routes& r, sharded<db::system_keyspace
         });
     });
 
+    cf::get_tombstone_gc.set(r, [&ctx] (const_req req) {
+        auto uuid = get_uuid(req.param["name"], ctx.db.local());
+        replica::table& t = ctx.db.local().find_column_family(uuid);
+        return !t.tombstone_gc_disabled();
+    });
+
+    cf::enable_tombstone_gc.set(r, [&ctx](std::unique_ptr<http::request> req) {
+        apilog.info("column_family/enable_tombstone_gc: name={}", req->param["name"]);
+        return foreach_column_family(ctx, req->param["name"], [](replica::table& t) {
+            t.set_tombstone_gc_disabled(false);
+        }).then([] {
+            return make_ready_future<json::json_return_type>(json_void());
+        });
+    });
+
+    cf::disable_tombstone_gc.set(r, [&ctx](std::unique_ptr<http::request> req) {
+        apilog.info("column_family/disable_tombstone_gc: name={}", req->param["name"]);
+        return foreach_column_family(ctx, req->param["name"], [](replica::table& t) {
```

Comment:
Shouldn't we have one API with a boolean parameter? We can keep using the same style as similar code if it's already this way elsewhere.

Reply:
This design mimics the enable/disable auto-compaction column_family api.
But I agree that it would be nice to add an optional boolean parameter to the POST interface that so that tombstone_gc can be disabled either by DELETE or by POST with e.g. `enabled=true`.

Reply:
Better to just mimic it and go no further.